identity-iota/src/tangle/client.rs

```diff
@@ -30,7 +30,7 @@ use crate::tangle::TangleRef;
 use crate::tangle::TangleResolve;
 
 /// Client for performing IOTA Identity operations on the Tangle.
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
`Clone` should not be on `Client`, either find a way to not do this or restrict it to the Wasm target only.